Makefile

```diff
@@ -160,6 +160,7 @@ cy-mobile-prod-run: web/inline_data_gen.go cypress
 
 web/src/schema.d.ts: graphql2/schema.graphql
 	go generate ./web/src
+	(cd web/src && yarn prettier -l --write schema.d.ts)
```

Comment:
Do this in the generate file rather than relying on it here. `go generate ./web/src` should result in the final file & formatting

Reply:
You can use `exec.LookupPath` (I think that was the one) but it would be easier to just add it to the generate file instead.

We are doing something similar in `graphql2/gen.go` where we are running `goimports` (our go formatter) after we generate the files. You can just add a second `go:generate` line that will call `yarn prettier` and should be good.

You could also call it directly instead of relying on yarn with `./node_modules/.bin/prettier` and that might be a bit safer/better.

Oh, and I forgot -- since we rely on prettier, add `web/src/node_modules` to as a dependency of `web/src/schema.d.ts` on line 161. That will make sure things are handled e.g. if you run `make clean` first (or just cloned the repo).